Gibe.AbTest/AbTestRepository.cs

```diff
@@ -17,31 +18,27 @@ public ExperimentDto GetExperiment(string id)
 		{
 			using (var db = _databaseProvider.GetDatabase())
 			{
-				return db.Single<ExperimentDto>("WHERE Id = @0", id);
+				return db.SingleOrDefault<ExperimentDto>("WHERE Id = @0", id);
 			}
 		}
 
-		public IEnumerable<ExperimentDto> GetExperiments()
+		public IEnumerable<ExperimentDto> GetEnabledExperiments()
 		{
 			using (var db = _databaseProvider.GetDatabase())
 			{
-				return db.Query<ExperimentDto>("FROM AbExperiment");
+				var now = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss");
+				return db.Fetch<ExperimentDto>($"FROM AbExperiment " +
+					$"WHERE [Enabled] = 1 " +
+					$"AND '{now}' >= [StartDate] OR StartDate IS NULL " +
+					$"AND '{now}' < [EndDate] OR StartDate IS NULL ");
```

Comment:
Would some brackets help disambiguate this (I always prefer being specific with a chain of `AND`/`OR` statements)
```suggestion
					$"(AND '{now}' >= [StartDate] OR StartDate IS NULL) " +
					$"(AND '{now}' < [EndDate] OR StartDate IS NULL)");
```